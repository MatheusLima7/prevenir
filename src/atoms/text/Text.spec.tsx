import React from "react";
import { render } from "@testing-library/react-native";

import Text from "./text";

describe("Text Component", () => {
  it("should render component correctly", () => {
    const container = render(
      <Text type="text" size="large">
        Teste
      </Text>
    );

    expect(container).toMatchSnapshot();
  });

  it("fontWeight attribute should return 400", () => {
    const { getByText } = render(
      <Text type="text" size="large">
        Teste
      </Text>
    );
    const textComponent = getByText("Teste");
    const style = textComponent.props.style;

    expect(style).toHaveProperty("fontWeight", "400");
  });
});
