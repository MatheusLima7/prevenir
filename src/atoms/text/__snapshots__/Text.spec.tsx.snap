// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Component should render component correctly 1`] = `
<Text
  size="large"
  style={
    {
      "color": undefined,
      "display": "flex",
      "fontFamily": "Quicksand-Regular",
      "fontSize": 24,
      "fontWeight": "400",
      "lineHeight": 30,
    }
  }
  testId="text-component"
  type="text"
>
  Teste
</Text>
`;
